frontend/src/studio-styles/typography.css

```
@@ -0,0 +1,184 @@
+  /* Code blocks */
+  :where(pre) {
+    display: flex;
+    background-color: var(--color-neutral-950);
+    border-radius: var(--radius-4xl);
+    overflow-x: auto;
+    margin-top: --spacing(10);
+    margin-bottom: --spacing(10);
+    margin-left: calc(-1 * (--spacing(6)));
+    margin-right: calc(-1 * (--spacing(6)));
+
+    @media (width >= theme(--breakpoint-sm)) {
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)